src/components/expanded-state/custom-gas/FeesGweiInput.js

```diff
@@ -29,14 +33,14 @@ const GweiStepButton = ({
   buttonColor,
 }) => {
   return (
-    <ButtonPressAnimation
+    <StepButtonWrapper
```

Comment:
wondering if is moving the +- buttons horizontally, i imagine it will at least move the distance between the input and buttons?

Reply:
yeah thats why I kept it at only 3px, we can either play with the numbers or use absolute positioning but I didn't want to mess with that since i'm not gonna be building android